### PR TITLE
Convert HTML to PDF with Docraptor

### PR DESCRIPTION
**Pull request by taves-hub**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #5084`

### CODE REVIEW

**Comment by taves-hub** [comment]
@mercymeave, how are you doing.  I have made the requested changes that were suggested by @lalith1403. You may have a look at it. It's a `duplicate`.

**Comment by taves-hub** [comment]
@lalith1403, all the requested changes committed. You may have a look at it. Thanks.

**Comment by lalith1403** [comment]
I had made a few changes in the earlier PR. please incorporate them as well.

**Comment by taves-hub** [comment]
Hello @lalith1403, I have incorporated them in this PR.

**Comment by mercymeave** [comment]
> Hello @lalith1403, I have incorporated them in this PR.

Taking a look

**Comment by kymomolly** [comment]
> > Hello @lalith1403, I have incorporated them in this PR.
> 
> Taking a look

@mercymeave, Thanks

**Comment by taves-hub** [comment]
> > Hello @lalith1403, I have incorporated them in this PR.
> 
> Taking a look

@mercymeave, Thanks.

**Review by mercymeave — CHANGES_REQUESTED** [review]
When you create a new PR, learn to incorporate the changes in the previous PR. Seems like you created a new file that ignored the changes I had made in the previously closed article.

**Comment by taves-hub** [comment]
@mercymeave, I have incorporated the changes you had made.

**Comment by hectorkambow** [comment]
> When you create a new PR, learn to incorporate the changes in the previous PR. Seems like you created a new file that ignored the changes I had made in the previously closed PR.

great point @mercymeave  

cc: @taves-hub lets be sure to go through the entire article to catch any issues that may have not been caught by our peer reviewer.

Our Peer Reviewer and Content Moderator teams do NOT provide any revisions services. All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner.

ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.

**Comment by taves-hub** [comment]
@hectorkambow, Hello sir, I double-checked my PR for grammatical errors and updated it. If you don't mind, I would like you to have a look at it. Thanks

**Comment by lalith1403** [comment]
Waiting for peer reviewers approval.

File: content/articles/convert-html-to-pdf-with-docraptor/index.md
Comment (mercymeave) [inline content/articles/convert-html-to-pdf-with-docraptor/index.md]: Where is the API key coming from?

File: content/articles/convert-html-to-pdf-with-docraptor/index.md
Comment (mercymeave) [inline content/articles/convert-html-to-pdf-with-docraptor/index.md:21]: What do you mean "simple joining" it is unclear.

File: content/articles/convert-html-to-pdf-with-docraptor/index.md
Comment (mercymeave) [inline content/articles/convert-html-to-pdf-with-docraptor/index.md:172]: Use a more clear heading, API documentation sounds more like a documentation of the API.

File: content/articles/convert-html-to-pdf-with-docraptor/index.md
Comment (mercymeave) [inline content/articles/convert-html-to-pdf-with-docraptor/index.md]: What do you mean by you get after sending an HTTP request? It is not clear.

**Review by mercymeave — CHANGES_REQUESTED** [review]
Hello @taves-hub,
Most of the changes @lalith1403  highlighted in the closed PR do not reflect here. You did not make the necessary changes.

File: content/articles/convert-html-to-pdf-with-docraptor/index.md
Comment (taves-hub) [reply content/articles/convert-html-to-pdf-with-docraptor/index.md:21]: @mercymeave,  I think that what I have tried to elaborate in the bracket

**Comment by taves-hub** [comment]
@mercymeave hello. I have incorporated all the changes. You can have a look and give guidance where necessary. Thank you.

cc
@lalith1403

**Comment by lalith1403** [comment]
@mercymeave Any updates?

**Comment by mercymeave** [comment]
> @mercymeave Any updates?

finalizing

**Review by mercymeave — APPROVED** [review]
Hello @lalith1403 

Would you please give this PR  a final review?

**Comment by kymomolly** [comment]
@lalith1403, hello sir. I was requesting if you can give this PR a final review since you had started it before closing the previous PR, which you were about to Merge. Thank you.

**Comment by taves-hub** [comment]
@lalith1403, hello sir. I was requesting if you can give this PR a final review since you had started the previous PR before closing it. Thank you.

**Comment by lalith1403** [comment]
Yes. I will review it by EOD

**Comment by hectorkambow** [comment]
jumping on this one

**Pull request merged**
